lib/autoupdate/agent/updater.go

```diff
@@ -749,7 +749,9 @@ func (u *Updater) LinkPackage(ctx context.Context) error {
 	} else if err != nil {
 		return trace.Errorf("failed to link system package installation: %w", err)
 	}
-	if err := u.Process.Sync(ctx); err != nil {
+	if err := u.Process.Sync(ctx); errors.Is(err, ErrNotSupported) {
+		u.Log.WarnContext(ctx, "SystemD is not installed, skip sync")
```

Comment:
```suggestion
		u.Log.WarnContext(ctx, "Systemd is not installed. Skipping sync.")
```
Just for consistency with the other logs. (SystemD is probably more correct.)